src/YouTube.js

```diff
@@ -76,16 +110,29 @@ const YouTubeComponent = ({ url, ...props }) => {
         setReady(true);
     };
 
+    const clickButton = useCallback(() => {
+        playerRef.current.internalPlayer.playVideo();
```

Comment:
You have to check `playerRef` reference before you use it

`if (!playerRef.current) return;`

It will throw an error if it's undefined